docs/building-modules/errors.md

```diff
@@ -38,7 +38,7 @@ execution.
 
 Example:
 
-+++ https://github.com/cosmos/cosmos-sdk/blob/v0.38.1/x/distribution/keeper/querier.go#L62-L80
++++ https://github.com/cosmos/cosmos-sdk/blob/7f59723d889b69ca19966167f0b3a7fec7a39e53/x/distribution/keeper/querier.go#L62-L80
```

Comment:
nit, I think the example should be from other code than the Querier as it will be deprecated